rtgui/editorpanel.cc

```diff
@@ -46,6 +46,8 @@ class EditorPanel::MonitorProfileSelector
     void prepareProfileBox ()
     {
         profileBox.set_size_request (100, -1);
+        profileBox.setPreferredWidth(90, 150);
```

Comment:
Even if you don't care, could you try to stay consistent, i.e. add a space before the parentheses of a function call? I'd say it is of similar importance as the trailing underscore.